.github/actions/spelling/line_forbidden.patterns

```diff
@@ -19,6 +24,12 @@
 # s.b. greater than
 \bgreater then\b
 
+# s.b. into
+#\sin to\s
+
+# s.b. opt-in
+#\sopt in\s
```

Comment:
Similarly, this isn't always right, and in fact the one case I ran into here was a bad example... (Some heuristics are really not good enough to leave lying around, and instead handling the associated cases should be left to a grammar checker that can be truly context aware...)